### ISSUE TITLE
Lots of NPE noise in language server

### ISSUE THREAD

**Issue by dsyer** [spring-projects/sts4#90]
This is apparently harmless (the project still compiles), but probably unintentional:

```
[pool-2-thread-1] ERROR org.springframework.ide.vscode.boot.java.utils.SpringIndexer - error extracting symbol information in project 'micro-function' - for docURI 'file:///home/dsyer/dev/thin/workspace/micro-function/src/main/java/com/example/app/Endpoint.java' - on node: @SuppressWarnings({"unchecked"}) @Bean public <T>RouterFunction<?> userEndpoints(){
  return route(POST("/"),request -> ok().body((Flux<T>)this.function.apply(request.bodyToFlux(this.inspector.getInputType(this.function))),(Class<T>)this.inspector.getOutputType(this.function)));
}

java.lang.NullPointerException
	at org.springframework.ide.vscode.boot.java.requestmapping.WebfluxPathFinder.visit(WebfluxPathFinder.java:52)
...
```

The project is https://github.com/dsyer/spring-boot-micro-apps.

**Comment by martinlippert** [comment]
I cloned the repo, but can't find this file:
`src/main/java/com/example/app/Endpoint.java` (to which the error message refers to)

Any hint where I can find that file/type?

**Comment by dsyer** [comment]
Sorry. There's a branch called "function". Hopefully you won't need a ton of other stuff that I only have locally.

**Comment by martinlippert** [comment]
I can see the code now, but can't reproduce the NPE. The NPE seems to be caused by the parser not being able to resolve a method binding, so maybe something else in the project or that file was broken at the same time?

Nevertheless the flux path finder should not throw a NPE in that case, but just ignore those situations and continue. I added a check for that, therefore marking this as solved for 4.0.1
